src/oc/web/components/ui/onboard_wrapper.cljs

```diff
@@ -364,12 +371,49 @@
              :class (when (:error org-editing) "error")
              :value (:name org-editing)
              :on-change #(dis/dispatch! [:input [:org-editing]
-                          (merge org-editing {:error nil :name (.. % -target -value)})])}]
+               (merge org-editing {:error nil :name (.. % -target -value)})])}]
+                 ;; Email domains row
+          [:div.org-email-domains-row.group
+            [:div.field-label
+              [:label
+                "Allowed email domain " [:span.info "(optional)"]]
+              (when (:error org-editing)
+                [:label.error
+                   "Only company email domains are allowed."])]
+            [:div.org-email-domain-field
+              {:class (when (:domain-error org-editing) "error")}
+              [:input.um-invite-field.email
+                {:name "um-domain-invite"
+                 :type "text"
+                 :auto-capitalize "none"
+                 :value (or (:email-domain org-editing) "")
+                 :pattern "@?[a-z0-9.-]+\\.[a-z]{2,4}$"
+                 :on-change #(let [domain (.. % -target -value)]
+                               (if (utils/valid-domain? domain)
+                                 (do
+                                   (dis/dispatch!
+                                    [:input [:org-editing :email-domain]
+                                     domain])
+                                   (dis/dispatch!
+                                    [:input [:org-editing :domain-error]
+                                     false]))
+                                 (do
+                                   (if (zero? (count domain))
+                                     (dis/dispatch!
+                                      [:input [:org-editing :domain-error]
+                                       false])
+                                     (dis/dispatch!
+                                      [:input [:org-editing :domain-error]
+                                       true]))
+                                   (dis/dispatch!
+                                    [:input [:org-editing :email-domain] nil]))))
```

Comment:
@thepug i don't think we need to reset this field in case of a non valid email... the UX is very bad, for example if i type opencompany.com and then a space or a comma to add another the check says it's not good and the value in the filed disappear completely.
Also i remain with the Create team button blocked because the domain-error flag is true but in fact i could post since the email domain is empty.